app/backend/app.py

```diff
@@ -28,27 +28,11 @@
 from approaches.readretrieveread import ReadRetrieveReadApproach
 from approaches.retrievethenread import RetrieveThenReadApproach
 
-# Replace these with your own values, either in environment variables or directly here
-AZURE_STORAGE_ACCOUNT = os.getenv("AZURE_STORAGE_ACCOUNT", "mystorageaccount")
```

Comment:
For ease of mocking the environment in tests, I moved these into the function below. This also means less globals, yay!